src/components/Footer.tsx

```diff
@@ -0,0 +1,97 @@
+import cn from 'classnames';
+import { useContext, useMemo } from 'react';
+
+import { FilterStatus } from '../types/FilterStatus';
+import { DispatchContext, StateContext } from './TodosProvider';
+import { deleteTodo } from '../api/todos';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+export const Footer = () => {
+  const { filteredBy, todos, newTodoInputRef } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+
+  const filteredStatus = Object.values(FilterStatus);
```

Comment:
Not mistake, but looks like no need to re create this variable on each render, so it should be moved out of component